db/migrate/20170203074044_create_surveys.rb

```diff
@@ -0,0 +1,10 @@
+class CreateSurveys < ActiveRecord::Migration[5.0]
+  def change
+    create_table :surveys do |t|
+      t.string :title
+      t.string :description
```

Comment:
It'll be good to validate columns on both the model level (which you've already done) AND at the database level. 
Validating at the *model* level: when user submits form, validation occurs without hitting the database. 
Validating at *database* level: ensures integrity of data. 
ie for Survey, you `validates_presence_of :title and :description`, do include `null: false` for title and description columns as well.